docs/tutorials/agent_framework/Chatbot_with_RAG.md

```diff
@@ -0,0 +1,318 @@
+{"index":{"_index":"test_tech_news"}}
+{"passage":"Amazon Bedrock is a fully managed service that offers a choice of high-performing foundation models (FMs) from leading AI companies like AI21 Labs, Anthropic, Cohere, Meta, Stability AI, and Amazon via a single API, along with a broad set of capabilities you need to build generative AI applications with security, privacy, and responsible AI. Using Amazon Bedrock, you can easily experiment with and evaluate top FMs for your use case, privately customize them with your data using techniques such as fine-tuning and Retrieval Augmented Generation (RAG), and build agents that execute tasks using your enterprise systems and data sources. Since Amazon Bedrock is serverless, you don't have to manage any infrastructure, and you can securely integrate and deploy generative AI capabilities into your applications using the AWS services you are already familiar with."}
+
+```
+## 1. Create LLM
+
+Follow step 2 "Prepare LLM" of [RAG_with_conversational_flow_agent](./RAG_with_conversational_flow_agent.md) to set up Bedrock Claude model.
+
+Note the model id, you will use it in following steps.
+
+## 2. Create Agent
+Create agent with `conversational` type. 
+
+Both `conversational_flow` and `conversational` agents can support conversation history.
```

Comment:
Both conversational_flow and conversational agents can support conversation history, but they differ in their operation: